### PR TITLE
chore: update usage for enabling session recording feature flag

### PR DESCRIPTION
**Pull request by marandaneto**
## :bulb: Motivation and Context
_#skip-changelog_

## :pencil: Checklist

- [ ] I reviewed the submitted code.
- [ ] I added tests to verify the changes.
- [ ] I updated the docs if needed.
- [ ] No breaking change or entry added to the changelog.

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -197,6 +197,8 @@ Enable `Record user sessions` on the [PostHog project settings](https://us.posth
 
 [Authorized Domains for Replay](https://us.posthog.com/settings/project-replay#replay-authorized-domains) has to be disabled, for now, or create a new testing project to not disrupt your current project configuration.
 
+Let us know your `organization_id` so we can enable the `session-replay-mobile` feature flag for your organization.
```
Comment (marandaneto) [inline USAGE.md:200]: @pauldambra how do I know the `organization_id`? checked the project config but didn't find it.

[the remark this answers is not in the document]
Comment (marandaneto) [reply USAGE.md:200]: @pauldambra what's about that? so I know what to do as well.

[the remark this answers is not in the document]
Comment (pauldambra) [reply USAGE.md:200]: LGTM

the quickest route I just found was jumping to the user in Django admin. That lists a "current organization". Clicking on that and the org id was in the URL

**Review by pauldambra — APPROVED** [review]
:shipit:

**Pull request merged**